# HG changeset patch
# User Sean Farley <sean@mcs.anl.gov>
# Date 1332269671 18000
#      Tue Mar 20 13:54:31 2012 -0500
# Node ID decbfb90b77a671e935547dd6916ee077c94f8f1
# Parent  42725dc9dd51805b60b432882ed6b24b82ed60c0
cmake: update 'install' target to also install gklib_tls.h

diff -r 42725dc9dd51 -r decbfb90b77a include/CMakeLists.txt
--- a/include/CMakeLists.txt	Tue Mar 20 13:54:30 2012 -0500
+++ b/include/CMakeLists.txt	Tue Mar 20 13:54:31 2012 -0500
@@ -1,6 +1,7 @@
 if(METIS_INSTALL)
   option (METIS_USE_LONGINDEX "Compile METIS with long ints (64bit integers)" OFF)
   option (METIS_USE_DOUBLEPRECISION "Compile METIS with double precision (double)" OFF)
   configure_file (metis.h.in metis.h)
   install(FILES "${PROJECT_BINARY_DIR}/include/metis.h" DESTINATION include)
+  install(FILES "${PROJECT_BINARY_DIR}/include/gklib_tls.h" DESTINATION include)
 endif()
